docs/keywords.md

```diff
@@ -0,0 +1,56 @@
+  # NL-Augmenter Keywords
+
+Every transformation should have keywords that identify the type of transformation that is being written. The following list can be used as reference.
+- the type of linguistic change that the transformation is attempting eg. morphological, lexical, syntactic, word-order, discourse, noise etc.
+- the type of algorithm that is written eg. rule-based, model-based, api-based, external-knowledge-based, transformer-based, parser-based, etc.
+- the naturalness of the generation eg. natural-sounding, natural-looking, unnatural, etc.
+- the potential accuracy & precision of the generation eg. accurate, near-accurate, high-precision, high-coverage, etc.
+- the text-specific modality being covered (if applicable) eg. visual, aural, written, etc.
+- the skills required to differentiate between the input and the output of the transformation (if applicable) eg. causal-reasoning, visual-reasoning, social-reasoning, humor, figurative-language, grammar, world-knowledge, etc. 
+- other transformation specific keywords
+
+Some of the categories can be subjective and they are meant to be used as best estimated. It is okay to include multiple of each type. Example, many transformer based generators might fit into most of the linguistic categories.  If your transformation is best described by keywords not yet included in the list below, please add the additional keywords to this file as part of your pull request.
+
+Keyword | Description
+------- | -----------
+**Type of Linguistic Change** |
+  `morphological` | character level or inflectional/morphological changes
```

Comment:
```suggestion
  `morphological` | character level or inflectional/morphological changes
```

In BIG-bench, I indented with unicode space characters, which are preserved during markdown display (see proposed edit to this line). You might want to consider doing something similar here, if you want the keywords to appear indented from the section headers. You can also use \&nbsp; if you want it to be more explicit.